packages/taiko-client/integration_test/README.md

```diff
@@ -1,4 +1,8 @@
 # How to debug test cases?
+- set up config
+```
+export L2_NODE=l2_geth
```

Comment:
By default which value will be used? `l2_reth`?

Reply:
This config is just for local debugging.
In CI/CD, they have different envs in different git workflows.

Reply:
`l2_geth ` is the default choice in `make test`.

Reply:
When we debug locally, we won't use `entrypoint.sh`, so there is no default value for our local debugging.

Reply:
Then let's set a default value in `entrypoint.sh`?

Reply:
And then we can avoid setting this value manually here

Reply:
We did not use `entrypoint.sh` in local debugging, so setting a default value in `entrypoint.sh` is not helpful.
We set up the test environment manually and just run the test case we want to debug.